### PR TITLE
docs: update the content of the FAQ document

### PR DESCRIPTION
**Pull request by Firstsawyou**
### What this PR does / why we need it:

Add content to the FAQ document.

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -341,3 +347,11 @@ Note this option is not shown in the output of `etcd --help`.
 ```
 
 Indeed this distinction was eliminated by etcd in their master branch, but not backport to announced versions, so be care when deploy your etcd cluster.
+
+## How to achieve high availability of APISIX?
+
+The high availability of APISIX can be divided into two parts:
+
+>1.Data plane: The data plane of Apache APISIX is stateless and can be elastically scaled at will. Just add a layer of LB in front.
```
Comment (spacewander) [inline FAQ.md:355]: As this word is not quoted from others, maybe we can use `1. Data...` directly?

Comment (Firstsawyou) [reply FAQ.md]: updated.

File: FAQ.md [hunk 2]
```diff
@@ -78,7 +78,9 @@ An example, `foo.com/product/index.html?id=204&page=2`, gray release based on `i
 1. Group A：id <= 1000
 2. Group B：id > 1000
 
-here is the way:
+There are two different ways to do this：
+
+1. Use the vars attribute of route to do it.
```
Comment (tokers) [inline FAQ.md:83]: `vars` is a field in the Route structure, please use backquotes to wrap it.

File: FAQ.md [hunk 3]
```diff
@@ -341,3 +347,11 @@ Note this option is not shown in the output of `etcd --help`.
 ```
 
 Indeed this distinction was eliminated by etcd in their master branch, but not backport to announced versions, so be care when deploy your etcd cluster.
+
+## How to achieve high availability of APISIX?
```
Comment (tokers) [inline FAQ.md:351]: What about "How to set up high available Apache APISIX clusters"

Comment (Firstsawyou) [reply FAQ.md]: updated.

File: FAQ.md (hunk 2 above)
Comment (Firstsawyou) [reply FAQ.md]: updated.

File: FAQ_CN.md [hunk 4]
```diff
@@ -296,7 +296,7 @@ etcd --enable-grpc-gateway --data-dir=/path/to/data
 
 事实上这种差别已经在 etcd 的 master 分支中消除，但并没有向后移植到已经发布的版本中，所以在部署 etcd 集群时，依然需要小心。
 
-## 如何实现 APISIX 的高可用？
+## 如何设置高可用的 Apache APISIX 集群？
```
Comment (tokers) [inline FAQ_CN.md:299]: It not "设置", you can use term like "创建" or “建立”.

Comment (Firstsawyou) [reply FAQ_CN.md]: updated.

**Review by spacewander — APPROVED** [review]

**Review by tokers — APPROVED** [review]

**Pull request merged**